custom_components/xiaomi_gateway3/core/devices.py

```diff
@@ -3130,6 +3130,18 @@
         BaseConv("backlight", "switch", mi="8.p.1"),
         BaseConv("led", "switch", mi="8.p.2"),
     ]
+}, {
+    # https://home.miot-spec.com/spec/linp.light.lp1bc
+    19653: ["Linptech", "Human Presence-Sensing Flat Panel Light", "LP1", "light.lp1bc"],
+    "spec": [
+        BaseConv("light", "light", mi="2.p.1"),
+        BrightnessConv("brightness", mi="2.p.2", max=100),
+        MapConv("mode", "select", mi="2.p.7", map={0: "None", 1: "Reading", 3: " Lighting", 4: "Night light"}),
+        MapConv("power_on_state", "select", mi="2.p.9", map={0: "default", 1: "on", 2: "off"}),
+        MapConv("occupancy_status", "sensor", mi="5.p.1", map={0: "NoOne", 1: "SomeOne"}),
```

Comment:
`BoolConv("occupancy", "binary_sensor", mi="5.p.1"),` might be better

Reply:
Change occupation to boolean type

![image](https://github.com/user-attachments/assets/bcf051ed-ccf9-4663-b5af-d0bc256b95e4)